-code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-34009/23258

ERROR: Build errors found during clang-tidy run.
```
gmake: *** [config/SCRAM/GMake/Makefile.coderules:129: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/RecoTracker/SpecialSeedGenerators/python/CombinatorialSeedGeneratorForCosmics_cfi.py
+++ b/RecoTracker/SpecialSeedGenerators/python/CombinatorialSeedGeneratorForCosmics_cfi.py
@@ -36,7 +36,7 @@ layerInfo = cms.PSet(
         clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutNone')),
         rphiRecHits = cms.InputTag("siStripMatchedRecHits","rphiRecHit"),
         maxRing = cms.int32(7)
-  )
+    )
 )
 combinatorialcosmicseedingtripletsTOB_layerList = cms.vstring('MTOB4+MTOB5+MTOB6',
     'MTOB3+MTOB5+MTOB6',
@@ -69,24 +69,26 @@ combinatorialcosmicseedfinder = cms.EDProducer("CtfSpecialSeedGenerator",
         GlobalY = cms.double(300.0)
     ),
     Charges = cms.vint32(-1),
-    OrderedHitsFactoryPSets = cms.VPSet(cms.PSet(
-        ComponentName = cms.string('GenericTripletGenerator'),
-        LayerSrc = cms.InputTag("combinatorialcosmicseedingtripletsTOB"),
-        PropagationDirection = cms.string('alongMomentum'),
-        NavigationDirection = cms.string('outsideIn')
-    ), 
-    cms.PSet(
-        ComponentName = cms.string('GenericPairGenerator'),
-        LayerSrc = cms.InputTag("combinatorialcosmicseedingpairsTECpos"),
-        PropagationDirection = cms.string('alongMomentum'),
-        NavigationDirection = cms.string('outsideIn')
-    ), 
-    cms.PSet(
-        ComponentName = cms.string('GenericTripletGenerator'),
-        LayerSrc = cms.InputTag("combinatorialcosmicseedingtripletsTIB"),
-        PropagationDirection = cms.string('oppositeToMomentum'),
-        NavigationDirection = cms.string('insideOut')
-    )),
+    OrderedHitsFactoryPSets = cms.VPSet(
+        cms.PSet(
+            ComponentName = cms.string('GenericTripletGenerator'),
+            LayerSrc = cms.InputTag("combinatorialcosmicseedingtripletsTOB"),
+            PropagationDirection = cms.string('alongMomentum'),
+            NavigationDirection = cms.string('outsideIn')
+        ), 
+        cms.PSet(
+            ComponentName = cms.string('GenericPairGenerator'),
+            LayerSrc = cms.InputTag("combinatorialcosmicseedingpairsTECpos"),
+            PropagationDirection = cms.string('alongMomentum'),
+            NavigationDirection = cms.string('outsideIn')
+        ), 
+        cms.PSet(
+            ComponentName = cms.string('GenericTripletGenerator'),
+            LayerSrc = cms.InputTag("combinatorialcosmicseedingtripletsTIB"),
+            PropagationDirection = cms.string('oppositeToMomentum'),
+            NavigationDirection = cms.string('insideOut')
+        )
+    ),
     UseScintillatorsConstraint = cms.bool(True),
     TTRHBuilder = cms.string('WithTrackAngle'),
     LowerScintillatorParameters = cms.PSet(

--- a/RecoTracker/SpecialSeedGenerators/python/CombinatorialSeedGeneratorForCosmics_cfi.py
+++ b/RecoTracker/SpecialSeedGenerators/python/CombinatorialSeedGeneratorForCosmics_cfi.py
@@ -53,9 +53,9 @@ combinatorialcosmicseedingpairsTECpos_layerList = cms.vstring('TEC1_pos+TEC2_pos
     'TEC7_pos+TEC8_pos',
     'TEC8_pos+TEC9_pos')
 combinatorialcosmicseedingtripletsTIB_layerList = cms.vstring('TIB1+TIB2+MTIB3')
+
 combinatorialcosmicseedfinder = cms.EDProducer("CtfSpecialSeedGenerator",
     SeedMomentum = cms.double(5.0), ##initial momentum in GeV !!!set to a lower value for slice test data
-
     ErrorRescaling = cms.double(50.0),
     RegionFactoryPSet = cms.PSet(
         RegionPSetBlock,
@@ -75,18 +75,18 @@ combinatorialcosmicseedfinder = cms.EDProducer("CtfSpecialSeedGenerator",
         PropagationDirection = cms.string('alongMomentum'),
         NavigationDirection = cms.string('outsideIn')
     ), 
-        cms.PSet(
-            ComponentName = cms.string('GenericPairGenerator'),
-            LayerSrc = cms.InputTag("combinatorialcosmicseedingpairsTECpos"),
-            PropagationDirection = cms.string('alongMomentum'),
-            NavigationDirection = cms.string('outsideIn')
-        ), 
-        cms.PSet(
-            ComponentName = cms.string('GenericTripletGenerator'),
-            LayerSrc = cms.InputTag("combinatorialcosmicseedingtripletsTIB"),
-            PropagationDirection = cms.string('oppositeToMomentum'),
-            NavigationDirection = cms.string('insideOut')
-        )),
+    cms.PSet(
+        ComponentName = cms.string('GenericPairGenerator'),
+        LayerSrc = cms.InputTag("combinatorialcosmicseedingpairsTECpos"),
+        PropagationDirection = cms.string('alongMomentum'),
+        NavigationDirection = cms.string('outsideIn')
+    ), 
+    cms.PSet(
+        ComponentName = cms.string('GenericTripletGenerator'),
+        LayerSrc = cms.InputTag("combinatorialcosmicseedingtripletsTIB"),
+        PropagationDirection = cms.string('oppositeToMomentum'),
+        NavigationDirection = cms.string('insideOut')
+    )),
     UseScintillatorsConstraint = cms.bool(True),
     TTRHBuilder = cms.string('WithTrackAngle'),
     LowerScintillatorParameters = cms.PSet(

